config/routes.rb

```diff
@@ -87,7 +87,10 @@
     end
     resources :teams
     resources :seasons
-    resources :conferences
+    resources :conferences, only: :show
```

Comment:
Can you add the `do … end` block of the following line to this one and remove lines 91..93? I think otherwise you'll lose the `only: :show`